app/src/main/java/com/example/dkirilova/gym/adapters/GymAdapter.java

```diff
@@ -152,5 +125,6 @@ public void setGyms(List<Gym> gyms) {
     public interface IGymAdapterController {
         void editOrDelete(Gym gym);
         void openDetails(Gym gym);
+        void setImage(ImageView ivImage, String image);
```

Comment:
how come the image is a string? is it maybe a path to a file instead?
the name is confusing